ibtk/include/ibtk/FEDataManager.h

```diff
@@ -395,6 +395,11 @@ class SubdomainToPatchLevelTranslation
  * it produces some artifacts at the coarse-fine interface, but that these
  * generally don't effect the overall solution quality.
  *
+ * <code>num_fischer_vectors</code>: The number of vectors used to compute
+ * initial guesses in FischerGuess. Defaults to 5. Using more vectors lowers the
```

Comment:
Is this enabled or disabled by default? If disabled, should this be something like "defaults to 5 when enabled"?

Reply:
It's enabled by default since the optimization helps a lot.